specification/maps/data-plane/Search/preview/2022-09-01-preview/geocoding.json

```diff
@@ -0,0 +1,872 @@
+    "/reverseGeocode/batch": {
+      "post": {
+        "description": "**Reverse Geocoding Batch API**\n\n\n**Applies to**: see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\n\n\nThe Reverse Geocoding Batch API sends batches of queries to [Reverse Geocoding API](https://docs.microsoft.com/en-us/rest/api/maps/search-v2/get-reverse-geocoding) using just a single API call. The API allows caller to batch up to **100** queries.\n>[!Important]\n>By using this feature, you agree to the preview legal terms. See the [Preview Supplemental Terms](https://azure.microsoft.com/en-us/support/legal/preview-supplemental-terms/) for additional details.\n\n### Submit Synchronous Batch Request\nThe Synchronous API is recommended for lightweight batch requests. When the service receives a request, it will respond as soon as the batch items are calculated and there will be no possibility to retrieve the results later. The Synchronous API will return a timeout error (a 408 response) if the request takes longer than 60 seconds. The number of batch items is limited to **100** for this API.\n```\nPOST https://atlas.microsoft.com/reverseGeocode/batch?api-version=2022-09-01-preview\n```\n### POST Body for Batch Request\nTo send the _reverse geocoding_ queries you will use a `POST` request where the request body will contain the `batchItems` array in `json` format and the `Content-Type` header will be set to `application/json`. Here's a sample request body containing 2 _reverse geocoding_ queries:\n\n\n```\n{\n  \"batchItems\": [\n    {\n      \"point\": \"-122.128275, 47.639429\"\n    },\n    {\n      \"point\": \"-122.341979399674, 47.6095253501216\"\n    }\n  ]\n}\n```\n\nA _reverse geocoding_ batchItem object can accept any of the supported _reverse geocoding_ [URI parameters](https://docs.microsoft.com/en-us/rest/api/maps/search-v2/get-reverse-geocoding#uri-parameters) except query.\n\n\nThe batch should contain at least **1** query.\n\n\n### Batch Response Model\nThe batch response contains a `summary` component that indicates the `totalRequests` that were part of the original batch request and `successfulRequests` i.e. queries which were executed successfully. The batch response also includes a `batchItems` array which contains a response for each and every query in the batch request. The `batchItems` will contain the results in the exact same order the original queries were sent in the batch request. Each item is of one of the following types:\n\n  - [`GeocodingResponse`](https://docs.microsoft.com/en-us/rest/api/maps/search-v2/get-reverse-geocoding#geocodingresponse) - If the query completed successfully.\n\n  - `Error` - If the query failed. The response will contain a `code` and a `message` in this case.\n\n\n",
+        "operationId": "Search_PostReverseGeocodingBatch",
+        "x-ms-examples": {
+          "A Reverse Geocoding Batch API call containing 2 Reverse Geocoding queries": {
+            "$ref": "./examples/ReverseGeocodingBatch.json"
```

Comment:
I'm thinking that maybe we can separate the example in the ReverseGeocodingBatch.json into 2 example files.
One is for 200 status code (all requests are good). Another is for 206 status code (partial requests are failed).
So that we can have some descriptions for the 206 example.

Please help to update the ForwardGeocodingBatch one. I should separate them before.

Reply:
Sure! I have separated the example into two files.
Forward GC: 
https://github.com/Azure/azure-rest-api-specs/pull/19951/commits/f09606ccfd655db2f1560cb37b08b7b88905f358
RecGC:
https://github.com/Azure/azure-rest-api-specs/pull/19951/commits/561eb50c5025c20bcd5af27a5892d897692757f8